### PR TITLE
V1.3 archive feature

### LINKED ISSUE #53: As a restaurant manager, I can archive employee contacts
... so that if an employee leaves and wishes to return in the future, I can easily restore their details.

### PR DESCRIPTION
**Pull request by huekoh**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ArchiveCommand.java
Comment (1rbg) [inline src/main/java/seedu/address/logic/commands/ArchiveCommand.java:1]: Straightforward implementation! LGTM.

File: src/main/java/seedu/address/model/person/Person.java
Comment (1rbg) [inline src/main/java/seedu/address/model/person/Person.java:1]: Should archiveStatus be shown in toString() method?

Comment (1rbg) [reply src/main/java/seedu/address/model/person/Person.java:1]: Compared in equals() method as well?

**Review by 1rbg — CHANGES_REQUESTED** [review]
Looks mostly good to me, except for the comments about the Person.java file. Code coverage can be improved in our next iteration.

File: src/main/java/seedu/address/model/person/Person.java
Comment (huekoh) [reply src/main/java/seedu/address/model/person/Person.java:1]: i think thats sounds good, missed out on that thanks for the catch.

Comment (huekoh) [reply src/main/java/seedu/address/model/person/Person.java:1]: will do so in the next iteration.

File: src/main/java/seedu/address/logic/commands/ArchiveCommand.java
Comment (huekoh) [reply src/main/java/seedu/address/logic/commands/ArchiveCommand.java:1]: nice!

**Review by 1rbg — APPROVED** [review]
LGTM, thanks for making the changes!

**Pull request merged**